### PR TITLE
ADEN-1780: Added new high impact slot

### PR DESCRIPTION
**Pull request by nandy-andy**
We don't have high impact slot like `INVISIBLE_SKIN` for mobile (Mercury). This is first try to have it :smile_cat:

### CODE REVIEW

File: front/styles/main/module/_ads.scss [hunk 1]
```diff
@@ -6,6 +6,10 @@
 	}
 }
 
+.ad-slot[id^="INVISIBLE_"] {
```
Comment (mklucsarits) [inline front/styles/main/module/_ads.scss]: I don't think this style declaration is a good idea. How about creating a new attribute, "hidden" or something", for the ad slot component that adds the CSS class `hide` to that ad? Then it doesn't matter what the ad's name or id is, and we wouldn't need such specific styles.

Comment (nandy-andy) [inline front/styles/main/module/_ads.scss]: Hi Matt! :smile_cat: I'm glad you asked! :sunny: 

This is a pattern copied from Oasis and I agree with the statement we shouldn't copy bad patterns but I think this was the only way here - it's "clean" and explicit. AdEngine removes `hidden` classes on slots once they're populated with an ad. Invisible slots are special case when we really do not want to make the slot visible (even if it contains an ad / a code from DFP).

**Comment by gabrys** [comment]
LGTM. Thanks

File: front/styles/main/module/_ads.scss (hunk 1 above)
Comment (mklucsarits) [inline front/styles/main/module/_ads.scss]: Thanks for the explanation.

File: front/templates/main/application.hbs [hunk 2]
```diff
@@ -10,6 +10,7 @@
 	{{smart-banner isVisible=view.smartBannerVisible}}
 	{{site-head smartBannerVisible=view.smartBannerVisible sideNavCollapsed=view.sideNavCollapsed trackClick='trackClick'}}
 	{{side-nav isCollapsed=view.sideNavCollapsed loadRandomArticle='loadRandomArticle'}}
+	{{ad-slot name='INVISIBLE_HIGH_IMPACT' noAds=noAds}}
```
Comment (hakubo) [inline front/templates/main/application.hbs]: I really like how we can add additional slots now :)

**Comment by hakubo** [comment]
LGTM

**Pull request merged**